nautobot/docs/installation/nautobot.md

```diff
@@ -269,10 +269,10 @@ At this point, we should be able to run Nautobot's development server for testin
 development instance:
 
 ```no-highlight
-$ nautobot-server runserver 0.0.0.0:8000 --insecure
+$ nautobot-server runserver 0.0.0.0:8080 --insecure
```

Comment:
Thanks for calling this out.  I debated on doing this but at the end of the day I decided it was the right thing to do.  Primarily because port 8080 is listed on the IANA port service name list https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml with this several firewalls already have a named rule for alt-http being on port 8080.  That said the production container that is built runs as a user in the container, not root, this is by design for security and the container can't listen on port 80 or 443, so we needed an alternate the nautobot user (within the container) could use.  We chose 8080 and 8443, for consistency we switched all environments to run on port 8080.